### ISSUE TITLE
Installation failing on OS X 10.10.5 with Homebrew

### ISSUE THREAD

**Issue by verdurin** [hakimel/reveal.js#1374]
I've installed Node and NPM via Homebrew:

```
brew info node
node: stable 4.1.1 (bottled), HEAD
Platform built on the V8 JavaScript runtime to build network applications
https://nodejs.org/
/usr/local/Cellar/node/4.1.1 (2715 files, 35M) *
  Poured from bottle
From: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/node.rb
==> Dependencies
Build: pkg-config ✔
Optional: openssl ✔
==> Options
--with-debug
        Build with debugger hooks
--with-full-icu
        Build with full-icu (all locales) instead of small-icu (English only)
--with-openssl
        Build with openssl support
--without-completion
        npm bash completion will not be installed
--without-npm
        npm will not be installed
--HEAD
        Install HEAD version
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d
```

When I try to run `npm install` from the full setup instructions, I see this error:

```
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 14.5.0
gyp ERR! command "/usr/local/Cellar/node/4.1.1/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/adam/src/github/reveal.js/node_modules/node-sass
gyp ERR! node -v v4.1.1
gyp ERR! node-gyp -v v3.0.1
gyp ERR! not ok
Build failed
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/Cellar/node/4.1.1/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v4.1.1
npm ERR! npm  v2.14.4
npm ERR! code ELIFECYCLE

npm ERR! node-sass@0.9.3 install: `node build.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node-sass@0.9.3 install script 'node build.js'.
npm ERR! This is most likely a problem with the node-sass package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node build.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls node-sass
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/adam/src/github/reveal.js/npm-debug.log
```

**Comment by verdurin** [comment]
Examples of the reported errors:

```
/Users/adam/.node-gyp/4.1.1/include/node/v8.h:214:13: note: candidate template ignored: could not match 'Local' against 'MaybeLocal'
  V8_INLINE Local(Local<S> that)
            ^
/Users/adam/.node-gyp/4.1.1/include/node/v8.h:326:13: note: candidate template ignored: could not match 'S *' against 'v8::MaybeLocal<v8::Object>'
  V8_INLINE Local(S* that)
            ^
In file included from ../binding.cpp:1:
../node_modules/nan/nan.h:777:26: error: no member named 'Use' in namespace 'node::Buffer'
    return node::Buffer::Use(v8::Isolate::GetCurrent(), data, size);
           ~~~~~~~~~~~~~~^
../node_modules/nan/nan.h:1809:12: error: no matching function for call to '_NanEscapeScopeHelper'
    return NanEscapeScope(NanNew(handle)->Get(kCallbackIndex)
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:499:43: note: expanded from macro 'NanEscapeScope'
# define NanEscapeScope(val) scope.Escape(_NanEscapeScopeHelper(val))
                                          ^~~~~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:495:27: note: candidate template ignored: substitution failure [with T = v8::Function]
  NAN_INLINE v8::Local<T> _NanEscapeScopeHelper(v8::Local<T> val) {
                          ^
../node_modules/nan/nan.h:1824:12: error: no matching function for call to '_NanEscapeScopeHelper'
    return NanEscapeScope(node::MakeCallback(
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:499:43: note: expanded from macro 'NanEscapeScope'
# define NanEscapeScope(val) scope.Escape(_NanEscapeScopeHelper(val))
                                          ^~~~~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:495:27: note: candidate template ignored: substitution failure [with T = v8::Value]
  NAN_INLINE v8::Local<T> _NanEscapeScopeHelper(v8::Local<T> val) {
                          ^
../node_modules/nan/nan.h:1897:12: error: no matching function for call to '_NanEscapeScopeHelper'
    return NanEscapeScope(handle->Get(NanNew(key)).As<v8::Object>());
           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:499:43: note: expanded from macro 'NanEscapeScope'
# define NanEscapeScope(val) scope.Escape(_NanEscapeScopeHelper(val))
                                          ^~~~~~~~~~~~~~~~~~~~~
../node_modules/nan/nan.h:495:27: note: candidate template ignored: substitution failure [with T = v8::Object]
  NAN_INLINE v8::Local<T> _NanEscapeScopeHelper(v8::Local<T> val) {
                          ^
../node_modules/nan/nan.h:2103:12: error: no member named 'IsExternalAscii' in 'v8::String'; did you mean 'IsExternal'?
  if (str->IsExternalAscii()) {
           ^~~~~~~~~~~~~~~
           IsExternal
/Users/adam/.node-gyp/4.1.1/include/node/v8.h:2174:8: note: 'IsExternal' declared here
  bool IsExternal() const;
       ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make: *** [Release/obj.target/binding/binding.o] Error 1
```

**Comment by verdurin** [comment]
I'm able to install node-sass when not inside the reveal.js directory:

```
temp1 adam$ ./node_modules/.bin/node-sass --version
node-sass	3.3.3	(Wrapper)	[JavaScript]
libsass  	3.2.5	(Sass Compiler)	[C/C++]
```

**Comment by verdurin** [comment]
I address this locally with #1375

**Comment by darribas** [comment]
Confirm this was happening to me on 10.10.5 as well just cloning from GitHub and the fix works. Thanks @verdurin!!!

**Comment by hakimel** [comment]
Fixed by #1375, thanks @verdurin!
